tests/test_codegen.py

```diff
@@ -58,6 +62,20 @@ def test_try_expect(self):
             sys.stdout.write(exc)""")
         self.assertAstSourceEqual(source)
 
+        source = textwrap.dedent("""\
+        try:
+            'spam'[10]
+        except IndexError as exc:
+            sys.stdout.write(exc)
+        else:
+            pass
+        finally:
+            pass""")
+        # This is interesting -- the latest 2.7 compiler seems to
+        # handle this OK, but creates an AST with nested try/finally
+        # and try/except, so the source code doesn't match.
+        self.assertAstSourceEqualIfAtLeastVersion(source, (3, 4), (1, 0))
```

Comment:
I don't really know the right way to do this.  The compiler _works_ on 2.7, by creating a nested try/except inside a try/finally.  If that was (2, 7) then we would be asserting a failure and the test would fail, because the round-tripped code out of the AST would contain these nested try statements.

I don't know when the combined behavior happened and am not really curious enough to install 7 flavors of Python to figure it out, so the (1, 0) basically says "Don't even bother to test below 3.4"